library/swift/src/filters/FilterTrailersStatus.swift

```diff
@@ -9,4 +9,14 @@ public enum FilterTrailersStatus<T: Headers>: Equatable {
   /// Calling `continueRequest()`/`continueResponse()` MUST occur when continued filter iteration
   /// is desired.
   case stopIteration(T)
+
+  /**
+   * Resume previously-stopped iteration, possibly forwarding headers and data, if iteration was
+   * previously stopped during an on*Headers or on*Data invocation.
+   *
+   * It is an error to return ResumeIteration if iteration is not currently stopped, and it is
+   * an error to include headers if headers have already been forwarded to the next filter
+   * (i.e. iteration was stopped during an on*Data invocation instead of on*Headers).
+   */
+  case resumeIteration(T, withHeaders: Headers? = nil, withData: Data? = nil)
```

Comment:
```suggestion
  case resumeIteration(headers: Headers? = nil, data: Data? = nil, trailers: T)
```

Reply:
This ordering matches Kotlin - I also think it's nice to name the values explicitly but don't feel too strongly